### PR TITLE
Command slcli vlan edit accept that we do not send any parameters

### PR DESCRIPTION
**Pull request by edsonarios**
Issue link #1708
```
softlayer-python/ (issue1708) $ slcli vlan edit 12345
Is required Name or Note or Tags
```

### CODE REVIEW

**Review by caberos — APPROVED** [review]
LGTM

File: SoftLayer/CLI/vlan/edit.py [hunk 1]
```diff
@@ -22,6 +23,9 @@
 def cli(env, identifier, name, note, tags):
     """Edit a vlan's details."""
 
+    if not (name or note or tags):
```
Comment (allmightyspiff) [inline SoftLayer/CLI/vlan/edit.py:26]: For more than 2 options, I prefer using [`any()`](https://docs.python.org/3/library/functions.html#any)  as I feel it is a little easier to read.
```suggestion
    if not any([name, note, tags]):
```

File: SoftLayer/CLI/vlan/edit.py [hunk 2]
```diff
@@ -22,6 +23,9 @@
...
+        raise exceptions.CLIAbort("Is required Name or Note or Tags")
```
Comment (allmightyspiff) [inline SoftLayer/CLI/vlan/edit.py:27]: "Name, Note or Tags is required" would be a more natural sounding sentence. But for this case, I'd prefer the following.

```suggestion
        raise exceptions.CLIAbort("At least one option is required")
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Just some minor style changes needed, thanks.

File: SoftLayer/CLI/vlan/edit.py (hunk 1 above)
Comment (edsonarios) [reply SoftLayer/CLI/vlan/edit.py:26]: Done

File: SoftLayer/CLI/vlan/edit.py (hunk 2 above)
Comment (edsonarios) [reply SoftLayer/CLI/vlan/edit.py:27]: Done

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**